pkgs/tools/X11/sx/default.nix

```diff
@@ -11,20 +18,28 @@ stdenv.mkDerivation rec {
     sha256 = "0xv15m30nhcknasqiybj5wwf7l91q4a4jf6xind8x5x00c6br6nl";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
-
   makeFlags = [ "PREFIX=$(out)" ];
 
-  postInstall = ''
-    wrapProgram $out/bin/sx \
-      --prefix PATH : ${lib.makeBinPath [ coreutils util-linux xorgserver xauth ]}
-  '';
+  solutions = {
+    sx = {
+      scripts = [ "bin/sx" ];
+      interpreter = "${bash}/bin/sh";
```

Comment:
Use `runtimeShell` instead of `bash`.

Reply:
Justification? We haven't really haggled this out yet.

I don't think I've ever seen any lucid documentation on the intended purpose of `runtimeShell` that would clarify whether it's appropriate here. 

The [upstream uses `/bin/sh`](https://github.com/Earnestly/sx/blob/2.1.7/sx#L1), so it seems odd to me to intentionally use `bin/bash` regardless of whether it's direct or through the `runtimeShell` abstraction.

Reply:
Yeah, I used `${bash}/bin/sh` explicitly here because `sh` is what upstream expects, not `bash`. Remember that this is not a simple symlink, it also activates the [Bash POSIX mode](https://www.gnu.org/software/bash/manual/html_node/Bash-POSIX-Mode.html).